runtime/compiler/env/J9SharedCache.hpp

```diff
@@ -69,6 +69,9 @@ class TR_J9SharedCache : public TR_SharedCache
 
    UDATA *rememberClass(J9Class *clazz, bool create=true);
 
+   UDATA rememberDebugCounterName(const char *name);
```

Comment:
it would have been more correct to use uintptrj_t here rather than UDATA and do the cast inside these functions, but I'll let it go as something easily cleaned up later if it the typing inconsistency becomes any kind of issue.